src/server.js

```diff
@@ -1 +1,24 @@
+var http = require('http'); //require is how you import a node core module
+var router = require('./router/router.js');
 
+//now we need to specify a host and a port, on a local server the host is allocated automatically. we also need to set up a port to make sure our server only "listens" to things coming through that port.
+
+var port = 4000; //port can be any number, usually 4 digits. just tells our server where to listen.
+
+var server = http.createServer(router);
+
+server.listen(port);
+
+console.log(`server up and running on port: ${port}`);
+
+// can navigate to this page with localhost:<your-port-here> in your browser
+
+// to make changes to the page produced by the server we much kill the server and rerun it
+
+// unless you use nodemon
```

Comment:
Ah, these are actually my original notes from the code-along! :sweat_smile: I should remove them completely, I believe this info is in the guide for mentors .md file